mozilla-release/tools/update-packaging/Makefile.in

```diff
@@ -36,10 +36,12 @@ full-update:: complete-patch
 ifeq ($(OS_TARGET), WINNT)
 MOZ_PKG_FORMAT	:= SFX7Z
 UNPACKAGE	= '$(subst $(DIST),$(ABS_DIST),$(INSTALLER_PACKAGE))'
+ifdef 0
 ifdef AB_CD
 UNPACKAGE	= '$(PACKAGE_BASE_DIR)/$(PACKAGE)'
 endif
 endif
+endif
```

Comment:
Need to add this to build normal .mar file for repacking (doens't work properly in our case, with AB_CD variable looking for installer file in wrong place)